src/main_gui.cpp

```diff
@@ -496,6 +501,7 @@ static Hotkey global_hotkeys[] = {
 	Hotkey('0' | WKC_ALT, "crash_game", GHK_CRASH),
 	Hotkey('1' | WKC_ALT, "money", GHK_MONEY),
 	Hotkey('2' | WKC_ALT, "update_coordinates", GHK_UPDATE_COORDS),
+	Hotkey('9' | WKC_ALT, "desync", GHK_FORCE_DESYNC),
```

Comment:
why 9, and not just 3? Feels a bit weird, to not pick the next number?

Reply:
Because the 9 key is next to 0 and desync next to crash is better than desync close to give money?

Reply:
I can live with that reasoning; but than it might help if we re-order this blob a bit to reflect that :) (and possibly even a comment to state that :D)

Reply:
@michicc 's argument is correct. I originally put it on 3 and then figured it was a potentially harmful key too close to much more harmless keys.